### DISCUSSION TITLE
5.2.0.0

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#335] [Announcements]
- Add border resistance
- Fix daemon crashes
- Reworked options
- UI for exclusion list

## Donation
If this project help you, you can give me a cup of coffee :) 
[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=YLGYPSHWTQ5UW&lc=FR&item_name=Mgth&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

**Full Changelog**: https://github.com/mgth/LittleBigMouse/compare/v5.1...v5.2

**Comment by MavicaCameras** [comment]
Program fails to load on two test machines. Shows in Task bar. But does not open in Windows 11.

**Reply by MavicaCameras** [reply]
Looking it loads in the task bar in the right. And that is the access point . TY.
